### ISSUE TITLE
Array extension via * throws errors

### ISSUE THREAD

**Issue by TheCleric** [Microsoft/pyright#68]
**Describe the bug**
When using the `*` operator to extend/unroll an array the following error is thrown:
`Unhandled expression type '<Expression>'`

**To Reproduce**
```python
test_array = ["123", "456"]

bigger_array = [*test_array, "789"]
```

**Expected behavior**
No error should be raised.

**VS Code extension or command-line**
VS Code

**Additional context**
Python 3.6 (Anaconda)

**Comment by erictraut** [comment]
Thanks for the bug report. I introduced a fix, and it will be in the next release.

**Comment by erictraut** [comment]
This is fixed in 1.0.11, which I just released.
